client/ayon_blender/plugins/publish/extract_usd.py

```diff
@@ -112,8 +112,7 @@ def get_attr_defs_for_instance(cls, create_context, instance):
             return []
         
         # Attributes logic
-        publish_attributes = instance["publish_attributes"].get(
-            cls.__name__, {})
+        publish_attributes = cls.get_attr_values_from_data(cls, instance.data)
```

Comment:
I didn't realize we're working with create instance here. This will probably work, but in that case it is not necessary to use `.data`.
```suggestion
        publish_attributes = cls.get_attr_values_from_data(cls, instance)
```